lib/features/registration/bloc/registration_event.dart

```diff
@@ -88,6 +88,20 @@ class RegistrationSignMandate extends RegistrationEvent {
   List<Object> get props => [guid, appLanguage];
 }
 
+class RegistrationStripeInit extends RegistrationEvent {
+  const RegistrationStripeInit();
+
+  @override
```

Comment:
minor - why do you need to override props if you do not add any new members in the class?

Reply:
Why? - Because I did not question it, i removed it now. 🙏